### ISSUE TITLE
Different versions of the pre-training weights

### ISSUE THREAD

**Issue by ChaojunZ** [xiaomoguhz/DeCLIP#9]
Thank you very much for taking the time to announce the different versions of the pre-training weights in your busy schedule. Could you please explain what the different names mean and what the differences are among them, such as the newly released one
DeCLIP2_EVA L_DINOv2 - L_csa_0. 1 _1. 0

**Comment by Kimsure** [comment]
I have reimplemented this work and may have some suggestions on your question. 'EVA' means using EVA-CLIP as a backbone, 'DINOv2' means using DINOv2 for context distillation, 'csa' means using the qq+kk attention at the last layer in CLIP, which remains the same setting in previous work SCLIP, and '0.1_1.0' means the coefficient weights of content and context distillation loss. I hope it can help you

**Comment by xiaomoguhz** [comment]
@Kimsure Thank you for your help in explaining！Apologies for the delayed response. DeCLIP2 represents our extended version, featuring further optimizations to the model architecture and the distillation process. We will detail these modifications in the extended version of the paper.@ChaojunZ
